pkg/utils/utils.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"gopkg.in/go-playground/validator.v9"
+	"math/rand"
+	"time"
+)
+
+var SeededRand *rand.Rand = rand.New(rand.NewSource(time.Now().UnixNano()))
+
+var Validate *validator.Validate
+
+func init() {
+	Validate = validator.New()
```

Comment:
what is this for?

Reply:
Validation of incoming requests. I've found out that unmarshalling JSON into struct will use default values where the proper field is not available - that seems :scream: to me but fortunately, there's a way how to validate the struct doesn't contain default values.  
Pretty powerful package, BTW.